pynest/examples/Potjans_2014/README.rst

```diff
@@ -1,61 +1,108 @@
+* ``helpers.py``: helper functions for network construction, simulation and evaluation
+* ``network_params.py``: network and neuron parameters
+* ``stimulus_params.py``: parameters for optional external stimulation
+* ``sim_params.py``: simulation parameters
+* ``reference_data``: reference data and figures obtained by executing ``run_microcircuit.py`` with default parameters
 
-   mpirun -n num_mpi_prc python example.py
+Running the Simulation
+######################
 
+By default, the variables ``N_scaling`` and ``K_scaling`` in ``network_params.py`` are set to
+`0.1` which is a good choice for running the microcircuit on a local machine.
+``N_scaling`` adjusts the number of neurons and ``K_scaling`` adjusts the indegrees.
+The full network can be run by setting these values to `1`.
+If this is done, the option to print the time progress should be switched off: ``'print_time': False`` in ``sim_params.py``.
 
-The default version of the simulation uses Poissonian input, which is defined
-in the file ``network_params.py`` to excite neuronal populations of the
-microcircuit. If no Poissonian input is provided, DC input is calculated, which
-should approximately compensate the Poissonian input. It is also possible to
-add thalamic stimulation to the microcircuit or drive it with constant DC
-input. This can be defined in the file ``stimulus_params.py``.
+To run the simulation, simply use:
 
-Authors
---------
+.. code-block:: bash
 
-Hendrik Rothe, Hannah Bos, Sacha van Albada
+   python run_microcircuit.py
 
+The output will be saved in the directory ``data``.
 
+The code can be `parallelized <https://nest-simulator.readthedocs.io/en/latest/guides/parallel_computing.html>`_ using OpenMP and MPI, if NEST has been built with these applications.
+The number of threads (per MPI process) can be chosen by adjusting ``local_num_threads`` in ``sim_params.py``.
+The command for running the script with two MPI processes would be:
+
+.. code-block:: bash
+
+   mpirun -n 2 python run_microcircuit.py
+
+Note on Parameters
+##################
+
+By default, the simulation uses external Poissonian input to excite all neuronal populations of the microcircuit, see ``poisson_input': True`` in ``network_params.py``.
+If set to ``False``, the Poissonian input is turned off and compensated approximately by calculated DC input.
+In addition to this ongoing external drive, a thalamic stimulation or a stimulation by an external DC input can be switched on in ``stimulus_params.py`` (the default for both types of stimuli is ``False``).
+
+The default random initialization of membrane voltages in this simulation uses population-specific means and standard deviations to reduce an initial activity burst in the network: ``'V_type': 'optimized'`` in ``network_params.py``.
+Previous implementations used the same mean and standard deviation for all populations which corresponds to setting ``'V_type': 'original'``.
```

Comment:
A bit confusing. Do you mean ` Previous implementations used the same mean and standard deviation for all populations by setting ``'V_type': 'original'`` `?

Reply:
add a comma: populations, which
I don't find this confusing, but maybe 'which is achieved by setting' would be even clearer?

Reply:
I changed it to ``, which is here achieved by setting``. In some previous implementations like the SLI version, the parameter ``V_type`` did not exist at all. In the previous PyNEST version the parameter existed with ``original```as default.